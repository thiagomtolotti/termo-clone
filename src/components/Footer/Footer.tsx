import styles from "./Footer.module.css";

export const Footer = () => {
  return <footer className={styles.footer}>Feito com ❤️ para Jessica</footer>;
};
