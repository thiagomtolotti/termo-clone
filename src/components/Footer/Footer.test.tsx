import { render } from "@testing-library/react";
import { Footer } from "./Footer";

describe("Footer component", () => {
  it("Should render correctly", () => {
    const { container } = render(<Footer />);

    expect(container).toMatchSnapshot();
  });
});
