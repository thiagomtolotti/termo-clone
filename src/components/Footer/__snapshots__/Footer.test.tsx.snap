// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component Should render correctly 1`] = `
<div>
  <footer
    class="footer"
  >
    Feito com ❤️ para Jessica.
  </footer>
</div>
`;
